src/lib/belenios/admin/countMaxVotesAndVoters.js

```diff
@@ -0,0 +1,13 @@
+function countMaxVotesAndVoters(votersList) {
+  const voters = JSON.parse(votersList);
+  const maxVoters = voters.length;
+  const maxVotes = voters.map(({ weight }) => weight).reduce(
+    (weightAcc, weight) => weightAcc + weight,
+  );
```

Comment:
Optimisation: Le map n'est pas nécéssaire tu passes 2 fois dans tes voters alors que tu pourrais simplement le faire une fois dans le reduce
```suggestion
  const maxVotes = voters.reduce(
    (weightAcc, { weight }) => weightAcc + weight,
    0,
  );
```

Reply:
![image](https://user-images.githubusercontent.com/26386976/143501043-efaa60d5-da28-4b5b-b7e6-3e83f85be608.png)